frontend/src/components/pages/RegistrantExperience/RegistrationSteps/Waiver/WaiverReducer.tsx

```diff
@@ -1,38 +1,19 @@
 import {
   WaiverInterface,
   OptionalClauseResponse,
-  RequiredClauseResponse,
   WaiverReducerDispatch,
   WaiverActions,
-  LoadedWaiver,
   ClickOptionalClause,
   FillDate,
   FillName,
-} from "../../../../types/waiverTypes";
-import { WaiverClause } from "../../../../types/AdminTypes";
+} from "../../../../../types/waiverTypes";
 
 const waiverReducer = (
   waiverInterface: WaiverInterface,
   action: WaiverReducerDispatch,
 ): WaiverInterface => {
   const newWaiverInterface: WaiverInterface = { ...waiverInterface };
   switch (action.type) {
-    case WaiverActions.LOADED_WAIVER: {
```

Comment:
I load waiver earlier (in the session selection step) so that we enforce earlier that the camp and waiver both have to exist by the time we reach the waiver step. It also means that we won't have the `LOAD_WAIVER` action since it will already exist by the time we're dealing with the waiver page